python/das_import.py

```diff
@@ -10,8 +10,12 @@ def do_das_query(query):
     Execute das_client for the specified query, and return parsed JSON output
     """
 
-    args = ['dasgoclient', '-json', '-format', 'json', '--query', query]
-    result = subprocess.check_output(args)
+    args = ['dasgoclient', '--format', 'json', '--query', query]
+
+    try:
+        result = subprocess.check_output(args)
+    except subprocess.CalledProcessError as exc:
+        result = exc.output
```

Comment:
I think it's safer to print a message that the query failed and exit, rather than work with the output of a failed query